icefoganalytics/elcc-data-management
# Fix Linting Setup in VSCode

# Context

I'm not sure I ever set up in an specific editor, just prettified/linted the whole project manually.
Currently the linting doesn't work properly as there is some kind of relative pathing problem.
i.e. the eslint file reads the tsconfig.json file, but can't find any of the files that are supposed to be included as lintable.

# Solution

Move linting to "per-service" architecture. Now `web` and `api` have their own custom linting setup.

# Testing Instructions

1. Boot VSCode.
2. Install the recommended plugins (newly added is Volar special eslint for vue)
3. Check that you get linting in the `api` folder.
4. Check that you get linting in the `web` folder.
